ios/KalturaPlayerViewManager.swift

```diff
@@ -554,7 +554,18 @@ class KalturaPlayerRNView : UIView {
     }
     
     func updateOttAnalyticsConfig(config: Dictionary<String, Any>) {
-        let ottAnalyticsConfig = AnalyticsConfig(params:config)
+        let baseUrl: String = config["baseUrl"] as? String ?? ""
+        
+        let timerInterval: Double = config["timerInterval"] as? Double ?? 0.0
+        
+        let ks: String = config["ks"] as? String ?? ""
+        
+        let partnerId: Int = config["partnerId"] as? Int ?? 0
+        
+        let isExperimentalLiveMediaHit: Bool = config["experimentalLiveMediaHit"] as? Bool ?? false
```

Comment:
What do we pass today from app?

Reply:
"ottAnalytics": {
			"partnerId": 5002,
			"baseUrl": "https://5002.frp1.ott.kaltura.com/api_v3/",
			"ks": "djJ8NTAwMnzMjM5IyF4euH4PpB36j5_468Hzz3pNp4Mbxh9Cn9w0fWyPaTxCpJuuHjlDPAX27Vpi6jqqVFXWOqJlJBfkejg_zQy9TLbh5AfE2Cixxm_Lv7528-1_vm8gweg2wpWUWhLv7vZsaGKcyQTuLABx6IHpMnmi9ADbhN50rC4HCLvshkleeNUDxuEyuZVrsM14LvknMzofcpv_qaqYm7tAcdz4Csr1pJRKRtxQELjWSfV7rh7gFcKf_IrJV5SQiMNK8FLFL9Xu3SFjZ-ku6HXCdeosOyMPmkK_92HXay5vKZx4Qqom7OX-rNZJSKQgDeCAaF_TW-HKlp55vLUFNm28Ofp_7nXDBAz6Q3BBJyhy9s0kNwes05ltnPsXYzNZ1-Qkl62Oy7J5YjR8PyZrFWYQ8pWhXRnadN6UZ87hrGo6uWntWg==",
			"timerInterval": 30,
			"experimentalLiveMediaHit": true,
			"epgId": 143162515
		}

Reply:
fixed